src/lib/snark_params/snark_params.ml

```diff
@@ -487,6 +497,33 @@ module Tick = struct
 
     let%test_unit "hash large number of chunks plus 2" =
       For_tests.run_hash_test ((Chunked_triples.Chunk.size * 250) + 2)
+
+    (* benchmark unchunked, chunked hashes *)
+
+    let%bench "hash one triple unchunked" = For_tests.hash_unchunked 1
+
+    let%bench_fun "hash one triple chunked" =
+      (* make sure chunk table deserialized *)
+      ignore (Lazy.force chunk_table) ;
```

Comment:
Everything that uses the chunk table should use the `bench_fun` thing. If you can run only one benchmark, or we change the ordering later we could get bogus results.

Reply:
OK